Doc/deprecations/pending-removal-in-future.rst

```diff
@@ -67,6 +67,9 @@ although there is currently no date scheduled for their removal.
   * ``EntryPoints`` tuple interface.
   * Implicit ``None`` on return values.
 
+* :mod:`logging`: the ``warn()`` method is deprecated, use ``warning()``
+  instead.
```

Comment:
Link to methods:
```suggestion
* :mod:`logging`: the :meth:`~logging.warn()` method is deprecated, use
  :meth:`~logging.warning()` instead.
```

Or also mention since when this was deprecated:

```suggestion
* :mod:`logging`: the :meth:`~logging.warn()` method has been deprecated
  since Python 3.3, use :meth:`~logging.warning()` instead.
```